From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Tue, 18 Aug 2020 05:15:35 +1200
Subject: backport: Use explode("\t") instead of Utils::parseTsvLine when
 importing organization TSV

Utils::parseTsvLine was only added in commit 20660b10f.
---
 webapp/src/Service/ImportExportService.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/webapp/src/Service/ImportExportService.php b/webapp/src/Service/ImportExportService.php
index 5b306f757..a11d012a5 100644
--- a/webapp/src/Service/ImportExportService.php
+++ b/webapp/src/Service/ImportExportService.php
@@ -626,7 +626,7 @@ class ImportExportService
         $l--; // No version line
         foreach ($content as $line) {
             $l++;
-            $line = Utils::parseTsvLine(trim($line));
+            $line = explode("\t", trim($line));
             if (count($line) !== 2) {
                 $message = sprintf('cannot parse organization on line %d: expected 2 tab-separated values, found %d',
                                    $l, count($line));
-- 
2.20.1

